### DISCUSSION TITLE
How to add default base styles for specific HTML elements?

### DISCUSSION THREAD

**Discussion by raf202** [unocss/unocss#3094] [Q&A]
Coming from Tailwind where I could do something like this:

```
@layer base {
  h1 {
    @apply text-2xl;
  }
}
```

I'm struggling to find something equivalent in Uno. The only thing I found are Preflights, but they only support raw CSS and not rules/custom rules. How can I add default base styles for specific HTML elements?

**Comment by chu121su12** [comment]
You can try `selector-x`. See test usage https://github.com/unocss/unocss/blob/c149294390cc9239dd6c78fb8661c7f53de1d3ff/test/assets/preset-uno-targets.ts#L44
